### DISCUSSION TITLE
Input outline color change

### DISCUSSION THREAD

**Discussion by ask4jm** [quasarframework/quasar#15408] [General - Components / Directives / etc]
![image](https://user-images.githubusercontent.com/5520266/217563703-c1a38ba1-6f9f-4a44-81a0-b0349922ed9b.png)

Need to change the blue rectangle outline that becomes visible whenever Input field is in focus to transparent. Any help?

Platform: Laravel 10-dev with breeze & tailwind.

Thanks & Regards

**Comment by MilosPaunovic** [comment]
Create a Codepen reproduction.

**Reply by ask4jm** [reply]
Thanks for the reply. Quasar is working fine in Codepen. Must be and issue with my laravel setup. Just asking if anyone else has experience it.

**Comment by suleimanas** [comment]
just change the color props

```
      <q-input color="grey-3" label-color="orange" outlined v-model="text" label="Label">
        <template v-slot:append>
          <q-icon name="event" color="orange" />
        </template>
      </q-input>
```

https://quasar.dev/vue-components/input#coloring

**Reply by ask4jm** [reply]
![image](https://user-images.githubusercontent.com/5520266/217857137-c2405cb5-dcaa-4865-bba8-e66e3734969c.png)
I mean the blue rectangle. Sorry.

**Reply by suleimanas** [reply]
Sorry we can't help with images :)
but that was added by tailwind, if you inspect the q-input styles you can see that tailwind overwrite some classes.

PS: you are using Quasar you don't need tailwind at all
